azure-pipelines.yml

```diff
@@ -66,50 +66,48 @@ jobs:
       python ./scripts/ci/test_index.py -v
     displayName: "Verify Extensions Index"
 
-# - job: TestsPython27
-#   displayName: "Tests Python 2.7"
-#   pool:
-#     vmImage: 'ubuntu-16.04'
-#   steps:
-#   - task: UsePythonVersion@0
-#     displayName: 'Use Python 2.7'
-#     inputs:
-#       versionSpec: 2.7
-#   - task: Bash@3
-#     displayName: "Tests Python 2.7"
-#     inputs:
-#       targetType: 'filePath'
-#       filePath: scripts/ci/test_source.sh
+- job: SourceTests
+  displayName: "Unittest Test, Build Test"
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6'
+      Python38:
+        python.version: '3.8'
+  steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python $(python.version)'
+      inputs:
+        versionSpec: '$(python.version)'
+    - bash: pip install wheel==0.30.0
+      displayName: 'Install wheel==0.30.0'
+    - bash: ./scripts/ci/test_source.sh
+      displayName: 'Run unittest and build test'
+      env:
+        ADO_PULL_REQUEST_LATEST_COMMIT: $(System.PullRequest.SourceCommitId)
+        ADO_PULL_REQUEST_TARGET_BRANCH: $(System.PullRequest.TargetBranch)
+
+- job: LintModifiedExtensions
+  displayName: "CLI Linter on Modified Extensions"
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+    - task: UsePythonVersion@0
+      displayName: 'Use Python 3.6'
+      inputs:
+        versionSpec: 3.6
```

Comment:
Is it intentional to remove 3.7 and add 3.6?

Reply:
No.
Python 3.6 takes 50%+ market share, Python 3.7 takes 12%+.
Besides, we actually put faith on Python version is independent as it should be.